src/hooks/data/concentrators/useConcentratorVaults.ts

```diff
@@ -92,9 +93,10 @@ export function useConcentratorVault({
   const compounderQuery = useContractRead({
     ...useRegistryContract(),
     functionName:
-      concentratorTargetAsset === "auraBAL"
+      concentratorTargetAsset === "0x616e8BfA43F920657B3497DBf40D6b1A02D4608d"
```

Comment:
This is lazy but I'm hoping we get the new registry methods implemented so we can clean all of this up.